src/asmcnc/production/z_head_qc_jig/z_head_qc_aftr_apr_21.py

```diff
@@ -786,8 +787,7 @@ def update_status_text(self, dt):
         self.consoleStatusText.text = self.sm.get_screen('home').gcode_monitor_widget.consoleStatusText.text
 
     def do_reboot(self):
```

Comment:
Why have this function in the first place? just call console_utils.reboot() directly

Reply:
Please call console_utils directl<